### PR TITLE
feat!: Remove unused brand colour tokens

### PR DESCRIPTION
**Pull request by VincentSmedinga**
# Describe the pull request

Thank you for contributing to the project!
Please use this template to help us handle your PR smoothly.

## What

Removes all brand colour tokens we don’t currently use.

## Why

Because they get published. We don’t want people to use them. We do want to limit the sizes of our published files.

## How

Removed them from the JSON object, backed them up in a text file in the repository.

## Checklist

Before submitting your pull request, please ensure you have done the following. Check each checkmark if you have done so or if it wasn't necessary:

- [x] Add or update unit tests
- [x] Add or update documentation
- [x] Add or update stories
- [x] Add or update exports in index.\* files
- [x] Start the PR title with a Conventional Commit prefix, [as explained here](https://github.com/Amsterdam/design-system/blob/main/documentation/publishing.md?plain=1#L11).

## Additional notes

- None

### CODE REVIEW

File: proprietary/tokens/src/brand/ams/color.tokens.json [hunk 1]
```diff
@@ -3,126 +3,50 @@
...
-          "20": { "value": "#E4EFFA" },
-          "30": { "value": "#C6DDF5" },
-          "40": { "value": "#A1CAF0" },
-          "50": { "value": "#72B5EB" },
           "60": { "value": "#009DE6" },
-          "70": { "value": "#008CCE" },
-          "80": { "value": "#007AB2" },
-          "90": { "value": "#006391" },
-          "100": { "value": "#004667" }
+          "80": { "value": "#007AB2" }
```
Comment (alimpens) [inline proprietary/tokens/src/brand/ams/color.tokens.json:7]: We currently don't use this value, should we add it? YAGNI and all... Same for blue 20 and the other '80' values except blue and red.

File: proprietary/tokens/src/brand/ams/color.tokens.json.txt [hunk 2]
```diff
@@ -0,0 +1,130 @@
+{
```
Comment (alimpens) [inline proprietary/tokens/src/brand/ams/color.tokens.json.txt:1]: This feels like it should belong in an internal wiki type of place, not the repo

File: proprietary/tokens/src/brand/ams/color.tokens.json (hunk 1 above)
Comment (VincentSmedinga) [reply proprietary/tokens/src/brand/ams/color.tokens.json]: Oh wow, I forgot to remove quite a bunch of them.

File: proprietary/tokens/src/brand/ams/color.tokens.json.txt (hunk 2 above)
Comment (VincentSmedinga) [reply proprietary/tokens/src/brand/ams/color.tokens.json.txt]: Alright, removed it.

**Review by alimpens — APPROVED** [review]

**Pull request merged**